data/mods/My_Sweet_Cataclysm/sweet_comestibles.json

```diff
@@ -74,7 +74,7 @@
     "name": "cookie",
     "symbol": "%",
     "container": "null",
-    "description": "Sweet and delicious cookies, just like grandma used to bake.  They shiver under your touch.  Weird!",
+    "description": "Sweet and delicious sugar cookies, just like grandma used to make.  They shake slightly in your hand in a mildly disturbing way when you look directly at them.",
```

Comment:
The first sentence must stay the same as the one in vanilla.

I don't really see what this change adds, having the second sentence be short makes the difference between those cookies and the vanilla ones a little bit more subtle to notice.